packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -301,6 +303,38 @@ export abstract class AbstractMessageManager<
     this.setMessageStatus(messageId, status);
   }
 
+  /**
+   * Updates the message rawData without changing the status.
+   *
+   * @param messageId - The id of the Message to sign.
+   * @param rawData - The data to update rawData in the message.
+   */
+  updateMessageDataInline(messageId: string, rawData: string) {
+    const message = this.getMessage(messageId);
+    /* istanbul ignore if */
+    if (!message) {
+      return;
+    }
+    message.rawData = rawData;
+    this.updateMessage(message);
+  }
+
+  /**
+   * Updates the message error without changing the status.
+   *
+   * @param messageId - The id of the Message to sign.
+   * @param error - The data to update error in the message.
+   */
+  updateMessageErrorInline(messageId: string, error: string) {
```

Comment:
I don't think this method is necessary. We stopped using the `errored` status in the original three message managers but the public key one still uses it. But even in the original code you're replacing, I don't think the `error` property was actually used at all. We just reject the promise with a generic error. I'd recommend doing the same here, at least for now.